rest/model/permissions.go

```diff
@@ -0,0 +1,16 @@
+package model
+
+import (
+	"github.com/evergreen-ci/evergreen"
+)
+
+type APIPermissions struct {
+	ProjectPermissions []APIPermission `json: "projectPermissions"`
+	DistroPermissions  []APIPermission `json: "distroPermissions"`
+}
+
+type APIPermission struct {
+	Key    string                 `json:"key"`
+	Name   string                 `json:"name"`
+	Levels []evergreen.Permission `json:"levels"`
```

Comment:
should we also have an API version of a single permission?

Reply:
the `evergreen.Permission` struct includes the json encoding that would be included in a single API permission struct. The logic for transforming the permissions data exists in `globals.go`, which is why there is no API version of a single permission. Does that answer your question?

Reply:
sgtm!